wsd/DocumentBroker.cpp

```diff
@@ -1146,11 +1146,21 @@ void DocumentBroker::startRenameFileCommand()
 
     blockUI("rename"); // Prevent user interaction while we start renaming.
 
+    const auto it = _sessions.find(_renameSessionId);
```

Comment:
This is needed because this is the session that started renaming. We could hold its session, but that seems problematic (we'd be extending its lifetime artificially). Instead, we just look up if that session still exists when we are ready to upload.